### PR TITLE
no service worker is built if flag is passed

### LINKED ISSUE #420: --service-worker=false still generates service worker
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
A sw.js file is still generated when `--service-worker=false` is passed.

**If the current behavior is a bug, please provide the steps to reproduce.**
Run build with `--service-worker=false`

**What is the expected behavior?**
A sw.js file will not be generated.

**Please mention other relevant information.**
- node version
v8.7.0
- npm version
v5.4.2
- Operating system
Ubuntu 17.10

### PR DESCRIPTION
**Pull request by prateekbh**
**What kind of change does this PR introduce?**

bugfix

**Did you add tests for your changes?**
Nope

**Summary**
After this PR `--no-service-worker` flag will not even build the service worker hence lesser confusion

**Does this PR introduce a breaking change?**
Nope

---

Closes #420

### CODE REVIEW

**Comment by prateekbh** [comment]
@lukeed  done

File: src/lib/webpack/webpack-client-config.js [hunk 1]
```diff
@@ -149,15 +157,13 @@ function isProd(config) {
 					/polyfills(\..*)?\.js$/,
 					/\.map$/,
 					/push-manifest\.json$/,
-					/.DS_Store/,
-					/\.git/
```
Comment (lukeed) [inline src/lib/webpack/webpack-client-config.js]: Oops, sorry, we wanna keep this guy #494

**Comment by lukeed** [comment]
I forgot I can do it too, will push the final fix real quick~

**Comment by lukeed** [comment]
Thanks @prateekbh 😄

**Pull request merged**